herddb-core/src/main/java/herddb/core/TableManager.java

```diff
@@ -1324,6 +1342,20 @@ public void onTransactionCommit(Transaction transaction, boolean recovery) throw
             forceFlushTableData = true;
         }
 
+        if (!transaction.lastSequenceNumber.after(bootSequenceNumber)) {
```

Comment:
Can we create a patch only for this single change? This way we will isolate the problem.
This branch is mixing too many fixed.
How does this sounds to you?

Reply:
Moved the patch on top of master, waiting for CI